src/test/unit/controllers/caseEventGlossaryController.test.ts

```diff
@@ -0,0 +1,37 @@
+import sinon from 'sinon';
+import { Response } from 'express';
+import CaseEventGlossaryController from '../../../main/controllers/CaseEventGlossaryController';
+import fs from 'fs';
+import path from 'path';
+import {CaseEventGlossaryService} from '../../../main/service/caseEventGlossaryService';
+import {mockRequest} from '../mocks/mockRequest';
+
+
+const rawData = fs.readFileSync(path.resolve(__dirname, '../mocks/CaseEventGlossary.json'), 'utf-8');
+const caseEventGlossaryData = JSON.parse(rawData);
+const caseEventGlossaryController = new CaseEventGlossaryController();
+
+const stub = sinon.stub(CaseEventGlossaryService.prototype, 'generateCourtEventGlossaryObject').returns(caseEventGlossaryData);
+
+describe('Status Description Controller', () => {
+  it('should render the status description page', () =>  {
+    const i18n = {
+      'case-event-glossary': {},
+    };
+
+    stub.withArgs(1).returns(caseEventGlossaryData);
+
+    const response = { render: function() {return '';}} as unknown as Response;
+    const request = mockRequest(i18n);
+    request.query = {courtId: '1'};
+
+    const responseMock = sinon.mock(response);
+
+    responseMock.expects('render').once().withArgs('case-event-glossary');
```

Comment:
this test is wrong and should be failing as you are not calling with those args

Reply:
this test is fixed now.